README.md

```diff
@@ -24,5 +24,83 @@
+
+## Status
+
+This project is currently considered experimental.
+
+## Why apisix-go-plugin-runner
+
+Apache APISIX offers many full-featured plugins covering areas such as authentication, security, traffic control, serverless, analytics & monitoring, transformations, logging.
+
+It also provides highly extensible API, allowing common phases to be mounted, and users can use these API to develop their own plugins.
+
+This project is APISIX Go side implementation that supports writing plugins in Go.
+
+Currently, Go Plugin Runner is provided as a library. This is because the convention of Go is to compile all the code into an executable file. 
+
+Although there is a mechanism for Go Plugin to compile the plugin code into a dynamic link library and then load it into the body. But as far as experience is concerned, there are still some imperfections that are not so simple and direct to use.
+
+The structure of the apache/apisix-go-plugin-runner repository on GitHub is as follows:
+
+```
+.
+├── cmd
+├── internal
+├── pkg
+```
+
+Internal is responsible for the internal implementation, `pkg` displays the external interface, and `cmd` provides examples of the demonstration.
+There is a subdirectory of `go-runner` under the `cmd` directory. By reading the code in this section, you can learn how to use Go Plugin Runner in practical applications.
+
+## How it Works
+
+At present, the communication between Go Plugin Runner and Apache APISIX is based on Unix socket's RPC. So Go Plugin Runner and Apache APISIX need to be deployed on the same machine.
+
+### Enable Go Plugin Runner
+
+As mentioned earlier, Go Plugin Runner is managed by Apache APISIX, which runs as a child process of APISIX. So we have to configure and run this Runner in Apache APISIX.
+
+The following configuration process will take the code `cmd/go-runner` in the `apisix-go-plugin-runner` project as an example.
+
+1. Compile the sample code. Executing make build generates the executable file go-runner.
+2. Make the following configuration in the conf/config.yaml file of Apache APISIX:
+
+```yaml
+ext-plugin:
+  cmd: ["/path/to/apisix-go-plugin-runner/go-runner", "run"]
+```
+
+With the above configuration, Apache APISIX pulls up `go-runner` when it starts and closes `go-runner` when it stops.
+
+In view of the fact that `apisix-go-plugin-runner` is used in the form of a library in the actual development process, you need to replace the above example configuration with your own executable and startup instructions.
+
+Finally, the startup Apache APISIX, `go-runner` will be started along with it.
+
+### Other configuration methods
+
+Of course, if you need to take these three steps every time you verify the functionality in the development process, it is quite tedious. So we also provide another configuration that allows apisix-go-plugin-runner to run independently during development.
+
+The Internal of apisix-java-plugin-runner
```

Comment:
Why java?

Reply:
I have a problem with why we do not set `APISIX_CONF_EXPIRE_TIME=3600`?

Reply:
> Why java?

When filling in the content, I referred to the Java-related and forgot to delete it :( 
Now have deleted it

Reply:
> `APISIX_CONF_EXPIRE_TIME`

It will be set automatically by APISIX